src/ophyd_async/tango/core/_signal.py

```diff
@@ -48,7 +48,7 @@ def tango_signal_rw(
     timeout: float = DEFAULT_TIMEOUT,
     name: str = "",
 ) -> SignalRW[SignalDatatypeT]:
-    """Create a `SignalRW` backed by 1 or 2 Tango Attribute/Command
+    """Create a `SignalRW` backed by 1 or 2 Tango Attribute/Command.
 
     Parameters
```

Comment:
Docstring isn't converted

Reply:
I was going to leave Tango for another day...